docs/support_files/api_reference.md

```diff
@@ -59,6 +59,24 @@ Multiple `AfterAll` hooks are executed in the **reverse** order that they are de
 
 ---
 
+#### `AfterStep([options,] fn)`
+
+Defines a hook which is run after each step.
+
+* `options`: An object with the following keys:
+  * `tags`: String tag expression used to apply this hook to only specific scenarios. See [cucumber-tag-expressions](https://docs.cucumber.io/tag-expressions/) for more information.
+  * `timeout`: A hook-specific timeout, to override the default timeout.
+* `fn`: A function, defined as follows:
+  * The first argument will be an object of the form `{sourceLocation: {line, uri}, result: {duration, status, exception?}, pickle}`
```

Comment:
The argument is quite old. Can we update this with what we actually pass in?

Reply:
Updated.